config/kdump.py

```diff
@@ -90,3 +95,110 @@ def kdump_num_dumps(db, kdump_num_dumps):
     check_kdump_table_existence(kdump_table)
 
     db.cfgdb.mod_entry("KDUMP", "config", {"num_dumps": kdump_num_dumps})
+    echo_reboot_warning()
+
+
+# 'remote' command ('sudo config kdump remote ...')
+@kdump.command(name="remote", short_help="Enable or Disable Kdump Remote")
+@click.argument('action', required=True, type=click.Choice(['enable', 'disable'], case_sensitive=False))
+@pass_db
+def kdump_remote(db, action):
+
+    """Enable or Disable Kdump Remote Mode"""
+    kdump_table = db.cfgdb.get_table("KDUMP")
+    check_kdump_table_existence(kdump_table)
+    current_remote_status = kdump_table.get("config", {}).get("remote", "false").lower()
+
+    if action.lower() == 'enable' and current_remote_status == 'true':
+        click.echo("Error: Kdump Remote Mode is already enabled.")
+        return
+
+    elif action.lower() == 'disable' and current_remote_status == 'false':
+        click.echo("Error: Kdump Remote Mode is already disabled.")
+        return
+
+    if action.lower() == 'disable':
+        ssh_string = kdump_table.get("config", {}).get("ssh_string", None)
+        ssh_key = kdump_table.get("config", {}).get("ssh_key", None)
+        if ssh_string or ssh_key:
+            click.echo("Error: Remove SSH_string, SSH_key from Config DB before disabling Kdump Remote Mode.")
```

Comment:
Why should the SSH keys removed before KDUCMP global disable?

Reply:
SSH keys needs to be removed before disabling Kdump Remote Mode to avoid leftover values in the configuration files. If not removed, these keys will stay in the "kdump-tools" file and could cause confusion or security issues later. Clearing them ensures the configuration is accurate and clean